axiom/client.py

```diff
@@ -180,8 +180,8 @@ def ingest(
         self,
         dataset: str,
         payload: bytes,
-        contentType: ContentType,
-        enc: ContentEncoding,
+        contentType: ContentType | None = None,
+        enc: ContentEncoding | None = None,
```

Comment:
Sure? They are mandatory, right?

Reply:
you are right, I thought we provided default values for those.

Reply:
but we have only a check:

```python
        # check if passed content type and encoding are correct
        if not contentType:
            raise ValueError("unknown content-type, choose one of json,x-ndjson or csv")

        if not enc:
            raise ValueError("unknown content-encoding")
```

Reply:
looks like its good now 👍